mobile_app_connector/models/match_user.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
+#    @author: Sylvain Laydernier <sly.laydernier@yahoo.fr>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import api, models
+from odoo.addons.base.ir.ir_mail_server import MailDeliveryException
+
+
+class MatchUser(models.AbstractModel):
```

Comment:
I think that this class is not useful. Why not integrating the logic directly in the controller? Furthermore, the `skip_password_reset` option is there but never used, I think you can simplify a bit the code by removing unecessary options.